hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -117,12 +130,19 @@ public static Path getArchiveLogPath(String archiveFolder) {
   }
 
   public void loadInstantDetailsInMemory(String startTs, String endTs) {
-    loadInstants(startTs, endTs);
+    setInstants(loadInstants(startTs, endTs));
   }
 
   public void loadCompletedInstantDetailsInMemory() {
-    loadInstants(null, true,
+    final List<HoodieInstant> instants = loadInstants(null, true,
         record -> HoodieInstant.State.COMPLETED.toString().equals(record.get(ACTION_STATE).toString()));
+    setInstants(instants);
```

Comment:
I'm shocked that this method does nothing before, the return value is ignored, please help to conform this @nsivabalan :)

Reply:
Oops, my fault, ignore this comment, the `readCommits` is updated in method `readCommit`.